commands/proofs/tx.go

```diff
@@ -22,6 +22,10 @@ data to other peers as needed.
 }
 
 func doTxQuery(cmd *cobra.Command, args []string) error {
+	if err := commands.RequireInit(cmd); err != nil {
```

Comment:
There is a problem, as some commands (eg. init and keys) do not require Init.

I tried to add it to the query root command via PreRunE, but that didn't work, as you noticed in the comment above.  If you have a better idea, I am totally game.  It hurts me a bit to copy this all over....

Only other idea was to have a decorator, that takes RunE and returns a RunE with this prefix. But I thought this was too much magic from the python world.  but happy to do.